examples/bitbucket/secret.yaml

```diff
@@ -0,0 +1 @@
+../role-resources/secret.yaml
```

Comment:
Just like the pipeline, I'd prefer if each example that used a secret created and used its own.